### PR TITLE
Update docker to node9 and update protobufjs

### PR DESCRIPTION
**Pull request by elbuo8**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "author": "José F. Romaniello <jfromaniello@gmail.com> (http://joseoncode.com)",
   "license": "MIT",
   "engines": {
-    "node": ">=0.10.40"
+    "node": ">=4.2.3"
```
Comment (siacomuzzi) [inline package.json]: is this ok?

File: server.js [hunk 2]
```diff
@@ -167,51 +162,47 @@ BaaSServer.prototype._handler = function (socket) {
...
     });
   }
 
   log.debug(sockets_details, 'connection accepted');
 
   const decoder = RequestDecoder();
 
-  decoder.on('error',  () => {
+  decoder.on('error',  (err) => {
     log.info(sockets_details, 'unknown message format');
```
Comment (siacomuzzi) [inline server.js]: why `err` is not part of the log?

**Comment by siacomuzzi** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (elbuo8) [reply package.json]: Shouldn't be running with older versions.

File: server.js (hunk 2 above)
Comment (elbuo8) [reply server.js]: Wasn't before 🤔 let me include it.

**Pull request closed without merging**